sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -474,24 +474,26 @@ public static ExtractKeyPhrasesResultCollection toExtractKeyPhrasesResultCollect
                     new HashMap<>();
                 if (!CoreUtils.isNullOrEmpty(documentEntities.getRelations())) {
                     documentEntities.getRelations().forEach(healthcareRelation -> {
-                        final HealthcareEntity targetEntity =
-                            healthcareEntities.get(getHealthcareEntityIndex(healthcareRelation.getTarget()));
-                        final HealthcareEntity sourceEntity =
-                            healthcareEntities.get(getHealthcareEntityIndex(healthcareRelation.getSource()));
-                        final HealthcareEntityRelationType relationType =
-                            HealthcareEntityRelationType.fromString(healthcareRelation.getRelationType());
-
-                        final Map<HealthcareEntity, HealthcareEntityRelationType> targetRelatedEntity =
-                            entityRelationMap.getOrDefault(targetEntity, new HashMap<>());
-                        targetRelatedEntity.putIfAbsent(sourceEntity, relationType);
-                        entityRelationMap.putIfAbsent(targetEntity, targetRelatedEntity);
-
-                        if (healthcareRelation.isBidirectional()) {
-                            final Map<HealthcareEntity, HealthcareEntityRelationType> sourceRelatedEntity =
-                                entityRelationMap.getOrDefault(sourceEntity, new HashMap<>());
-                            sourceRelatedEntity.putIfAbsent(targetEntity, relationType);
-                            entityRelationMap.putIfAbsent(sourceEntity, sourceRelatedEntity);
-                        }
+                        // TODO: bidirectional relation is gone, do it in another PR
+                        // issue: https://github.com/Azure/azure-sdk-for-java/issues/19180
```

Comment:
I really appreciate how you are keeping this PR small and you are going to address design issues in other PRs